app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -88,12 +88,12 @@ private void setSession(@NonNull Session session) {
         this.languageResId = session.lang != null ? decideLanguageResId(session.lang.toUpperCase()) : R.string.lang_en;
     }
 
-    public Maybe<Session> findSession(int sessionId) {
+    public Completable findSession(int sessionId) {
         final String languageId = Locale.getDefault().getLanguage().toLowerCase();
-        return sessionsRepository.find(sessionId, languageId)
-                .map(session -> {
+        return  sessionsRepository.find(sessionId, languageId)
+                .flatMapCompletable(session -> {
                     setSession(session);
```

Comment:
I think this `finsSession` should return `sessions` though it's not used now.
Maybe this `setSession(session)` make confused to us.
How about changing this method name to `findAndSetSession()` or `initSession()`?
Hmm, I didn't decide good name... Sorry 🙇